sw/airborne/modules/datalink/telemetry_common.h

```diff
@@ -60,7 +60,7 @@ struct telemetry_cb_slots {
  *  and the list of registered callbacks
  */
 struct periodic_telemetry {
-  uint8_t nb;                     ///< number of messages
+  uint16_t nb;                    ///< number of messages
```

Comment:
this is the maximum number of messages to be registered, not related to the actual message ID.